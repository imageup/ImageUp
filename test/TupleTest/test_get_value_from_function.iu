function adder(int : x) -> int
{
	return (x + 1);
}

function multiply_f(float : y) -> float
{
	return (y * 2.0);
}

function main() -> int
{
	int : a = 0;
	tuple : tu = (0, adder(a));
	printf(tu[0]);
	printf(tu[1]);
	prints("correct output should be: 0, 1");

	float : b = 2.0;
	tuple : tu1 = (3, multiply_f(b) * 2.0);
	printf(tu1[0]);
	printf(tu1[1]);
	prints("correct output should be 3, 8");

	return 0;
}