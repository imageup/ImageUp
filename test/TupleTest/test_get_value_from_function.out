0
1
correct output should be: 0, 1
3
8
correct output should be 3, 8
